modele-social/règles/bénéficiaire.yaml

```diff
@@ -0,0 +1,142 @@
+bénéficiaire:
+  valeur: oui
+  description: |
+    Un bénéficiaire est un actionnaire dans une SAS ou un associé dans une SARL/EURL.
+
+    Attention: nous ne prenons en compte ici que le cas de figure de l'associé unique (SASU et EURL).
+
+# [TODO] [dividendes-indep]
+bénéficiaire . compte courant d'associé:
+  par défaut: 0€
+  titre: Sommes versées en compte courant d'associé
+  question: Quelles sont les sommes versées en compte courant par l'associé ?
+  description: >
+    Cette valeur est nécessaire à calculer le montant maximal imposable au PFU
+    pour un bénéficiaire au régime des travailleurs indépendants.
+
+    Le montant pris en compte est le solde moyen annuel du compte courant. Il
+    est déterminé par la somme des soldes moyens mensuels du compte, divisée par
+    le nombre de mois compris dans l’exercice.
+
+    Le solde moyen mensuel est égal à l’addition des soldes journaliers, divisée
+    par le nombre de jours dans le mois. Le solde moyen mensuel diffère donc du
+    solde mensuel figurant dans les comptes de la société.
+
+    En cas d’ouverture ou de clôture du compte en cours d’exercice, le nombre de
+    mois au cours de l’exercice sera réduit au nombre de mois de fonctionnement
+    du compte. Un compte est considéré comme « fonctionnant », même s’il n’est
+    pas mouvementé au cours de l’exercice.  La date à laquelle les sommes
+    versées en compte courant doivent être appréciées est le dernier jour de
+    l’exercice précédant le versement des intérêts.
+  références:
+    Circulaire RSI C2014-001: https://www.secu-independants.fr/uploads/tx_rsirss/C2014-001.pdf
+
+bénéficiaire . dividendes:
```

Comment:
@mquandalle

Ici j'ai décidé de mettre `valeur: non` pour m'assurer que les questions relatives aux dividendes n'apparaissent pas dans les autres simulateurs existants. Je trouve ça assez bancal de mettre une telle considération dans la base de règles. Une autre approche, plus basique mais plus fastidieuse dans ce cas, serait de mettre `bénéficiaire . dividendes : non` dans toutes les autres configs.

Reply:
Justement, `par défaut: non` ne donne pas l'effet souhaité. Dans le simulateur SASU, on aura:

<img width="588" alt="image" src="https://user-images.githubusercontent.com/1884912/131993861-3095e317-3f7f-4775-b946-5878eacea3e5.png">

Ca revient donc à devoir modifier config du simulateur SASU pour explicitement exclure la question `bénéficiaire . dividendes`. On retombe sur la question de mon commentaire: exclure tout d'un coup avec `valeur: non` (et faire un opt-in comme je l'ai fait dans la config dividendes.yaml), ou bien mettre `par défaut: non` et mettre un opt-out dans tous les simulateurs qui font intervenir l'impôt (à revoir plus en détail).